coverage-tests/overrides.js

```diff
@@ -0,0 +1,90 @@
+    'check frame after manual switch to frame with vg classic': {skip: true},			//diffs
+    'check region by native selector': {skip: true},				                //Will be implement in separate task
+    'check region by selector in frame in frame fully with scroll stitching': {skip: true},	//diffs
+    'check hovered region by element with css stitching': {skip: true},				//diffs
+    'check hovered region by element with scroll stitching': {skip: true},			//OutOfBoundsError: Region [Region(67, 0, 191 x 29, CONTEXT_RELATIVE)] is out of screenshot bounds [Region(0, 0, 685 x 460, SCREENSHOT_AS_IS)]
+    'check region by selector in overflowed frame after manual scroll with css stitching': {skip: true},	//Stale element
+    'check region by selector in overflowed frame after manual scroll with scroll stitching': {skip: true},	//Stale element
+    'check regions by coordinates in frame with css stitching': {skip: true},		//Unable to locate element: {"method":"css selector","selector":"#modal2"}
+    'check regions by coordinates in frame with scroll stitching': {skip: true},	//Unable to locate element: {"method":"css selector","selector":"#modal2"}
+    'check regions by coordinates in frame with vg': {skip: true},			//diffs
+    'should find regions by visual locator': {skip: true},				//Visual locators not implemented
+    'should find regions by visual locator with vg': {skip: true},			//Visual locators not implemented
+    'should not check if disabled': {skip: true},					//Unable to locate element: {"method":"css selector","selector":"[id="someId"]"}
+    'should hide and restore scrollbars with scroll stitching': {skip: true},		//diff
+    'should hide and restore scrollbars with vg': {skip: true},				// Unable to locate element: {"method":"css selector","selector":"#inner-frame-div"}
+    'should send floating region by coordinates in frame with vg': {skip: true},	//diffs
+    'should not send dom': {skip: true},						//diffs - info["actualAppOutput"][0]["image"]["hasDom"] == False,    assert True == False
+    'should send dom and location when check window with vg': {skip: true}, // not implemented in VG
+    'should send dom on ie': {skip: true},						//diffs - assert info["actualAppOutput"][0]["image"]["hasDom"] == True   assert False == True
+    'should send dom on edge legacy': {skip: true},					//eyes.open(driver)   -  EyesError: eyes.open_base() failed
+    'should set viewport size on edge legacy': {skip: true},				//EyesError: Failed to set the viewport size
+    'check window after manual scroll on safari 11': {skip: true},			//diffs
+    'check window after manual scroll with vg': {skip: true},				//diffs
+    'should send accessibility regions by selector with css stitching': {skip: true},		//Index error. [imageMatchSettings][accessibility] is empty...
+    'should send accessibility regions by selector with scroll stitching': {skip: true},	//Index error. [imageMatchSettings][accessibility] is empty...
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7812239)
```suggestion
    "should send accessibility regions by selector with scroll stitching": {skip: true},	//Index error. [imageMatchSettings][accessibility] is empty...
```